reference/4.0/Microsoft.PowerShell.Core/Out-Default.md

```diff
@@ -49,7 +49,7 @@ Accept wildcard characters: False
 ```
 
 ### -Transcript
-{{Fill Transcript Description}}
+
```

Comment:
Should have at least something for the description here instead of blank.

Reply:
fixed